.rubocop.yml

```diff
@@ -1,3 +1,9 @@
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
We could definitely use a newer version here (this used to be picked up from `.ruby-version`), but that generated new cop warnings so I decided against it.